README.md

```diff
@@ -1,37 +1,38 @@
 # FinOps Foundation SIG Containers
 
-Primary Authors: tbc
+Primary Authors: Mike Fuller
 
 Reviewed and Contributed to by: tbc
 
 # Introduction
 
-tbc
+SIG-Containers exists to explore how FinOps applies to the container world within cloud. To help identify/define the added cloud financial management challenges that containers introduce and solutions/best practices around how to apply FinOps to solve them. 
 
 # Mission Statement
 
-tbc
-
 1. **Clarify and inform.** Provide valuable and objective information to the TAC, End Users and Projects of the FinOps Foundation regarding areas considered in-scope. Strengthen the project ecosystem to meet the needs of end users and project contributors. Educate and inform users with unbiased, effective, and practically useful information.
 2. **Collaborate and interrelate.** Effectively interface with other related groups internal and external to the FinOps Foundation and connect the dots to facilitate meaningful collaborative progression of relevant topics. Engage more communities and create an on-ramp to effective TAC contribution &amp; recognition.
 3. **Assist and attract projects.** Helping to maintain the continued health of FinOps Foundation containers-related definitions. Focus attention and resources on helping foster project maturity, systematically across FinOps Foundation projects. Clarify relationship between projects, FinOps Foundation project staff, and community volunteers. Identify gaps in the landscape of FinOps Foundation containers definitions. Find and attract projects to fill these gaps through invitation to present, diligence on proposed projects, and in essence, act as a funnel for TAC project reviews in this area.
 4. **Afford impartial stewardship.** Provide and maintain a vendor neutral venue for relevant thought validation, discussion, and project feedback. Establish procedures to assist where necessary.
 
 ## In Scope
 
-TBC
+Containers within CSP managed container environments running on a CSP platform
+Containers within self-managed container environments running on a CSP platform
+
 
 ### Current FinOps Foundation Containers Working Groups
 
-TBC
+tbc
 
 ## Out of Scope
 
-Generally anything not considered in scope.
+Generally anything not considered in scope including containers running within private owned datacenters.
```

Comment:
See comment above. Is there a particular reason to exclude these? Unless you are fully cloud-native (which is relatively rare, based on data in the market as well as surveys we've undertaken).